packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -296,6 +297,31 @@ export interface BoxProps {
    * Content selection
    */
   userSelect?: "none" | "text" | "contain" | "all" | "auto";
+  /**
+   * Whether to apply the body text color class
+   * @default true
+   */
+  applyColor?: boolean;
+  /**
+   * Whether to apply the normal font weight class
+   * @default true
+   */
+  applyFontWeight?: boolean;
+  /**
+   * Whether to apply the normal font style class
+   * @default true
+   */
+  applyFontStyle?: boolean;
+  /**
+   * Whether to apply the base font size class
+   * @default true
+   */
+  applyFontSize?: boolean;
+  /**
+   * Whether to apply the base line height class
+   * @default true
+   */
+  applyLineHeight?: boolean;
```

Comment:
I'm concerned about this type of API see my comment in the PR

Reply:
@andrewberg-okta lets talk more at standup I'm sure we'll work something out thank you